contrib/kind.sh

```diff
@@ -766,6 +766,7 @@ install_ovn() {
   run_kubectl apply -f ovs-node.yaml
   run_kubectl apply -f ovnkube-master.yaml
   run_kubectl apply -f ovnkube-node.yaml
+  run_kubectl apply -f ovnkube-cluster-manager.yaml
```

Comment:
ovnkube-master will fail for a node if there are no subnets configured.  So it would keep retrying.

Does kind/k8s support the ordering ? But how to ensure this ordering ? given that there will be 1 cluster-manager pod and 'n' ovnkube-masters (where n = no of zones).